Workflow/Modules/ModuleBase.py

```diff
@@ -303,22 +303,14 @@ def _resolveInputStep( self ):
 
     self.stepName = self.step_commons['STEP_INSTANCE_NAME']
 
-    if 'executable' in self.step_commons and self.step_commons['executable']:
-      self.executable = self.step_commons['executable']
-    else:
-      self.executable = 'Unknown'
+    self.executable = self.step_commons.get('executable', 'Unknown')
```

Comment:
This is not strictly the asme thing, in case `step_commons['executable']` is `None` for example

Reply:
By construction this is either a string (not empty) or it doesn't exist.